performance_test/index.ts

```diff
@@ -0,0 +1,70 @@
+import { createLogger, format, transports } from "winston"
+import { connect } from "rabbitmq-stream-js-client"
+import { randomUUID } from "crypto"
+import { argv } from "process"
+import { PerfTestProducer } from "./perf_test_producer"
+import { inspect } from "util"
+
+const logger = createLogger({
+  level: "info",
+  format: format.combine(
+    format.colorize(),
+    format.timestamp(),
+    format.align(),
+    format.splat(),
+    format.label(),
+    format.printf((info) => `${info.timestamp} ${info.level}: ${info.message} ${info.meta ? inspect(info.meta) : ""}`)
+  ),
+  transports: new transports.Console(),
+})
+
+function parseArgs(args) {
+  const zip = (a, b) => Array.from(Array(Math.min(a.length, b.length)), (_, i) => [a[i], +b[i]])
```

Comment:
This is kinda hard to parse for me...
It's a bit more verbose, but maybe easier on the eyes?
const zip = <T, U>(a: T[], b: U[]): [T, U][] => {
  const shorterArray = a.length > b.length ? a : b
  const zipped = shorterArray.map((_, i) => [a[i], b[i]] as [T, U])
  return zipped
}